types/choose_sequence/index.js

```diff
@@ -90,10 +91,24 @@ module.exports = ({ card, tags }) => {
 
       var contentWrapper = document.querySelector('#${cardId}_wrapper');
       contentWrapper.addEventListener('click', function (ev) {
-        if (ev.target.tagName === 'BUTTON' && question.innerHTML.indexOf(emptyGuessMark) !== -1) {
-          question.innerHTML = question.innerHTML.replace(emptyGuessMark, ev.target.innerHTML);
-          ev.target.classList.add('selected-option');
-          ev.target.disabled = true;
+        if (ev.target.tagName === 'BUTTON') {
+          if (question.innerHTML.indexOf(emptyGuessMark) !== -1) {
+            var activeGuess = question.querySelector('.active-guess');
+            activeGuess.textContent = ev.target.textContent;
+            ev.target.classList.add('selected-option');
+            ev.target.disabled = true;
+          } else if (correctAnswersAmount === 1) {
```

Comment:
looks like I don't understand the logic - or button or `correctAnswersAmount === 1` ?